extensions/amp-story/1.0/story-analytics.js

```diff
@@ -28,6 +28,8 @@ export const StoryEventType = {
   PAGE_VISIBLE: 'story-page-visible',
   STORY_MUTED: 'story-audio-muted',
   STORY_UNMUTED: 'story-audio-unmuted',
+  TOOLTIP_CLOSED: 'story-tooltip-closed',
+  TOOLTIP_OPENED: 'story-tooltip-opened',
```

Comment:
Should we keep the `enter` - `exit` naming for consistency?
Also, is there any better name than `tooltip` for this? My point is that it's really not obvious for publishers to know that they can track clicks on their tweets using `TOOLTIP_ENTER`.

Additionally, publishers will want to know what the user clicked on: what URL? what tweet / instagram / youtube ID?
This would need to be tracked in the variable service.

This makes me wonder if this feature is actually much more complex than we thought:

- publishers will want to know if the tooltip was triggered, but for a link, do we actually want to send an event saying whether the user followed the link? Or do publishers track that themselves?
- for a Tweet or a YouTube ID, they want to know which ID. But how can they tell if what is clicked was a YouTube or Tweet embed? Do we need different events?